### ISSUE TITLE
Unique terrain features

### ISSUE THREAD

**Issue by ceymore** [andrewpey/open-voxel-tycoon#3]
I think unique terrain features as seen in Alpha Centauri can bring uniqueness to the random generated world. There we had several pre-defined locations which would get placed into the generated map - a volcano (or a very tall mountain here), a meteorite crater (basically a circular mountain chain), a large plateau with dense forest, a long mountain chain, a coral reef, etc..

I think those could be made adjustable (length of the mountain chain, height of the "volcano", size of the plateau). 

This really brought life to the standard random generated world in Alpha Centauri, as they brought contrast to the standard features we would see. I think it could be implemented here in a similar fashion.

**Comment by andrewpey** [comment]
Agree, we're considering to add biome features to the landscape generation,

I think this issue is a good place to suggest more such things (and discuss existant when it's ready).
